app/views/plantings/_thumbnail.html.haml

```diff
@@ -16,7 +16,10 @@
       %dt Finished on:
       %dd= "#{display_finished(planting)}"
       %dt Sun/shade?:
-      %dd= "#{display_sunniness(planting)}"
+      %dd
+        - sunniness = planting.sunniness.blank? ? "not specified" : planting.sunniness
```

Comment:
Hi cesy. This is all good. The file name of the icon for unspecified sunniness is "sunniness_not specified.png". The 'sunniness' variable will decide which icon will be shown. In the case of a blanked sunniness, the 'sunniness' variable will contain 'not specified' and the image_tag will load the image 'sunniness_not specified.png'